src/main/io/displayport_msp_bf_compat.c

```diff
@@ -592,9 +594,10 @@ uint8_t getBfCharacter(uint8_t ch, uint8_t page)
 
     case SYM_HUD_SIGNAL_4:
         return BF_SYM_HUD_SIGNAL_4;
-*/
+
     case SYM_HOME_DIST:
-        return BF_SYM_HOMEFLAG;
```

Comment:
This one is probably ok. Pin represeting home.

Reply:
Yep! It's the only available "home" icon in the font table too.